libs/core/src/lib/combobox/combobox.component.ts

```diff
@@ -335,6 +336,14 @@ export class ComboboxComponent
      */
     inShellbar = false;
 
+    fullWidth = false;
+
+    @Input()
+    showInput = true;
+
+    @Input()
+    disableHideShowOfInput = false;
```

Comment:
Same as above

Reply:
Right. Actually, according to design we should  allow user to enable/disable the hiding behaviour and to show directly opened/closed input. For that I should add inputs to shellbar component.